action.yml

```diff
@@ -16,6 +16,30 @@ inputs:
   environment-variables:
     description: 'Variables to add to the container. Each variable is of the form KEY=value, you can specify multiple variables with multi-line YAML strings.'
     required: false
+  environment-secrets:
```

Comment:
@joshmello can you think about how to add option to get secrets from AWS AppConfig? For example. we can set arn of config. It's will be very useful. Or just read env from file. 
(asking because have a lot of env to setup)

Reply:
by "environment-secrets" you mean both value from SSM and from Secret Manager right ?